src/Compute/Compute.AlcWrapper/AzureStorageService.cs

```diff
@@ -56,4 +60,25 @@ internal PSBlobDownloadInfo(BlobDownloadInfo info)
 
         public string ContentType { get { return _blobDownloadInfo.ContentType; } }
     }
+
```

Comment:
Line 28-62 should be removed, and rename the file name to be consistent with class name.

Reply:
let me know if the change i made is what you were looking for